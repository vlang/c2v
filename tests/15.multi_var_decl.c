int main() {
int n, r, sum = 0, temp;
}