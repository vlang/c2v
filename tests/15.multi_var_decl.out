[translated]
module main

fn main() {
	n := 0
	r := 0
	sum := 0
	temp := 0
}
